### PR TITLE
Update all translations

### PR DESCRIPTION
**Pull request by HichamELBSI**
#### Description of what you did:
Change content types to collection types in translations

#### My PR is a:

- [ ] 💥 Breaking change
- [ ] 🐛 Bug fix
- [ ] 💅 Enhancement
- [ ] 🚀 New feature

#### Main update on the:

- [ ] Admin
- [ ] Documentation
- [ ] Framework
- [ ] Plugin

#### Manual testing done on the following databases:

- [ ] Not applicable
- [ ] MongoDB
- [ ] MySQL
- [ ] Postgres
- [ ] SQLite

### CODE REVIEW

**Comment by derrickmehaffy** [comment]
@HichamELBSI your PR is failing the DCO check: https://github.com/strapi/strapi/pull/5103/checks

File: packages/strapi-admin/admin/src/translations/en.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
   "Analytics": "Analytics",
   "Content Manager": "Content Manager",
-  "Content Type Builder": "Content Type Builder",
+  "Content Type Builder": "Types Builder",
```
Comment (Aurelsicoko) [inline packages/strapi-admin/admin/src/translations/en.json]: We decided to keep the current name "Content-Types Builder". A Content-Type is a Collection or a Single Types, both together they are the Content-Types.

File: packages/strapi-admin/admin/src/translations/en.json [hunk 2]
```diff
@@ -32,7 +32,7 @@
   "app.components.HomePage.button.quickStart": "START THE QUICK START TUTORIAL",
   "app.components.HomePage.community": "Find the community on the web",
   "app.components.HomePage.community.content": "Discuss with team members, contributors and developers on different channels.",
-  "app.components.HomePage.create": "Create your first Content Type",
+  "app.components.HomePage.create": "Create your first Collection Type",
```
Comment (Aurelsicoko) [inline packages/strapi-admin/admin/src/translations/en.json]: Nop, you should keep "Content-Type"

File: packages/strapi-admin/admin/src/translations/en.json [hunk 3]
```diff
@@ -167,7 +167,7 @@
   "HomePage.helmet.title": "Homepage",
   "HomePage.welcome.congrats": "Congrats!",
   "HomePage.welcome.congrats.content": "You are logged as the first administrator. To discover the powerful features provided by Strapi,",
-  "HomePage.welcome.congrats.content.bold": "we recommend you to create your first Content-Type.",
+  "HomePage.welcome.congrats.content.bold": "we recommend you to create your first Collection-Type.",
```
Comment (Aurelsicoko) [inline packages/strapi-admin/admin/src/translations/en.json]: "Content-Type"

File: packages/strapi-admin/admin/src/translations/fr.json [hunk 4]
```diff
@@ -2,7 +2,7 @@
   "Analytics": "Statistiques",
   "Auth & Permissions": "Auth & Permissions",
   "Content Manager": "Content Manager",
-  "Content Type Builder": "Content Type Builder",
+  "Content Type Builder": "Types Builder",
```
Comment (Aurelsicoko) [inline packages/strapi-admin/admin/src/translations/fr.json]: Keep the old one

File: packages/strapi-admin/admin/src/translations/fr.json [hunk 5]
```diff
@@ -32,7 +32,7 @@
   "app.components.HomePage.button.quickStart": "VOIR LE QUICK START TUTORIEL",
   "app.components.HomePage.community": "Rejoignez la communauté",
   "app.components.HomePage.community.content": "Discutez avec les membres de l'équipe, contributeurs et développeurs sur différent supports.",
-  "app.components.HomePage.create": "Créez votre premier Content Type",
+  "app.components.HomePage.create": "Créez votre premier Collection Type",
```
Comment (Aurelsicoko) [inline packages/strapi-admin/admin/src/translations/fr.json]: "Créez votre première collection"

File: packages/strapi-admin/admin/src/translations/fr.json [hunk 6]
```diff
@@ -164,7 +164,7 @@
   "HomePage.helmet.title": "Accueil",
   "HomePage.welcome.congrats": "Bravo!",
   "HomePage.welcome.congrats.content": "Vous êtes connecté en tant que premier Administrateur. Afin de découvrir les fonctionnalités proposées par Strapi,",
-  "HomePage.welcome.congrats.content.bold": "nous vous conseillons de créer votre premier Type de Contenu.",
+  "HomePage.welcome.congrats.content.bold": "nous vous conseillons de créer votre premier Type de Collection.",
```
Comment (Aurelsicoko) [inline packages/strapi-admin/admin/src/translations/fr.json]: "votre première Collection"

File: packages/strapi-plugin-content-manager/admin/src/translations/fr.json [hunk 7]
```diff
@@ -1,6 +1,6 @@
 {
-  "models": "Types de Contenus",
-  "models.numbered": "Types de Contenus ({number})",
+  "models": "Types de Collection",
```
Comment (Aurelsicoko) [inline packages/strapi-plugin-content-manager/admin/src/translations/fr.json]: Keep the old one

File: packages/strapi-plugin-content-manager/admin/src/translations/fr.json [hunk 8]
```diff
@@ -92,13 +92,13 @@
...
-  "containers.SettingsPage.Block.contentType.title": "Types de contenu",
+  "containers.SettingsPage.Block.contentType.title": "Types de collection",
   "containers.SettingsPage.Block.generalSettings.description": "Configurez les options par défault de vos modèles",
   "containers.SettingsPage.Block.generalSettings.title": "Général",
   "containers.SettingsPage.pluginHeaderDescription": "Configurez les paramètres de vos modèles et groupes",
   "containers.SettingsView.list.title": "Paramètres d'affichage",
-  "containers.SettingsView.list.subtitle": "Configurez le layout et l'affichage de vos types de contenus et groupes",
-  "emptyAttributes.button": "Ouvrir le Content Type Builder",
+  "containers.SettingsView.list.subtitle": "Configurez le layout et l'affichage de vos types de collection et groupes",
+  "emptyAttributes.button": "Ouvrir le Types Builder",
```
Comment (Aurelsicoko) [inline packages/strapi-plugin-content-manager/admin/src/translations/fr.json]: "Ouvrir le constructeur de types de contenu"

File: packages/strapi-plugin-content-type-builder/admin/src/translations/en.json [hunk 9]
```diff
@@ -52,7 +52,7 @@
   "form.attribute.component.option.repeatable": "Repeatable component",
   "form.attribute.component.option.repeatable.description": "Best for multiple instances (array) of ingredients, meta tags, etc..",
   "form.attribute.component.option.reuse-existing": "Use an existing component",
-  "form.attribute.component.option.reuse-existing.description": "Reuse a component already created to keep your data consistent across content types.",
+  "form.attribute.component.option.reuse-existing.description": "Reuse a component already created to keep your data consistent across collection types.",
```
Comment (Aurelsicoko) [inline packages/strapi-plugin-content-type-builder/admin/src/translations/en.json]: content-types

File: packages/strapi-plugin-content-type-builder/admin/src/translations/fr.json [hunk 10]
```diff
@@ -33,8 +33,8 @@
   "form.attribute.item.uniqueField": "Champ unique",
   "form.attribute.item.uniqueField.description": "Vous ne pourrez pas créer une entrée s'il existe un champ similaire",
   "form.attribute.settings.default": "Valeur par défault",
-  "form.button.add.field.to.component": "Ajouter un nouveau champ à composant",
-  "form.button.add.field.to.contentType": "Ajouter un nouveau champ à ce type de contenu",
+  "form.button.add.field.to.component": "Ajouter un nouveau champ à ce composant",
+  "form.button.add.field.to.contentType": "Ajouter un nouveau champ à ce type de collection",
```
Comment (Aurelsicoko) [inline packages/strapi-plugin-content-type-builder/admin/src/translations/fr.json]: "à cette Collection"

File: packages/strapi-plugin-content-type-builder/admin/src/translations/fr.json [hunk 11]
```diff
@@ -45,12 +45,12 @@
...
   "menu.section.models.name.singular": "Modèle",
   "modelPage.attribute.relationWith": "Relation avec",
-  "modelPage.contentHeader.emptyDescription.description": "Il n'y a pas de description pour ce type de contenu",
+  "modelPage.contentHeader.emptyDescription.description": "Il n'y a pas de description",
   "plugin.description.long": "Modélisez la structure de données de votre API. Créer des nouveaux champs et relations en un instant. Les fichiers se créent et se mettent à jour automatiquement.",
   "plugin.description.short": "Modélisez la structure de données de votre API.",
   "popUpForm.navContainer.advanced": "Réglages avancés",
   "popUpForm.navContainer.base": "Réglages de base",
-  "popUpWarning.bodyMessage.contentType.delete": "Êtes-vous sûr de vouloir supprimer ce type de contenu ? Cela le supprimera aussi de vos types de contenu.",
+  "popUpWarning.bodyMessage.contentType.delete": "Êtes-vous sûr de vouloir supprimer ce type de collection ? Cela le supprimera aussi de vos types de contenu.",
```
Comment (Aurelsicoko) [inline packages/strapi-plugin-content-type-builder/admin/src/translations/fr.json]: "cette Collection"

File: test/config/front/testUtils/commonTrads.json [hunk 12]
```diff
@@ -1,7 +1,7 @@
 {
   "Analytics": "Analytics",
   "Content Manager": "Content Manager",
-  "Content Type Builder": "Content Type Builder",
+  "Content Type Builder": "Types Builder",
```
Comment (Aurelsicoko) [inline test/config/front/testUtils/commonTrads.json]: Keep the previous value

File: packages/strapi-admin/admin/src/translations/fr.json (hunk 4 above)
Comment (HichamELBSI) [reply packages/strapi-admin/admin/src/translations/fr.json]: Without a dash between "Content" and "Type"?

**Comment by Aurelsicoko** [comment]
LGTM

**Pull request merged**